scripts/build.py

```diff
@@ -163,3 +163,8 @@ def get_cache_slug():
     CSS_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', 'css', '')
     CSS_OUT_DIR = os.path.join(BUILD_DIR, 'css', '')
     copy_files_source_to_target(CSS_SRC_DIR, CSS_OUT_DIR)
+
+    # Copy core/templates/prod/head/ to build/[cache_slug]/templates/head/
+    TEMPLATES_SRC_DIR = os.path.join('core', 'templates', 'prod', 'head', '')
+    TEMPLATES_OUT_DIR = os.path.join(BUILD_DIR, 'templates', 'head', '')
```

Comment:
If we don't ever actually use core/templates/prod/head, can we skip the intermediate step of copying stuff to core/templates/prod/head and then copying that stuff to the build directory? Or is the intermediate step needed in order to run the dev server with minified scripts?

This is another case, I think, where the prod/build workflow in the scripts really needs to be refactored; it's getting convoluted.